Vagrantfile

```diff
@@ -11,17 +12,20 @@ Vagrant.configure("2") do |config|
     ]
   end
 
-  config.vm.define "appserver"
+  config.vbguest.auto_update = false
 
-  #Port forwarding
-  # NOTE: there could be a different script that sets the resolv.conf and then
+  ## Network configurations ##
+  config.vm.hostname = "appserver.test"
+  config.vm.network :private_network, ip: "192.168.168.168"
+  ## Port forwarding
+  #NOTE: there could be a different script that sets the resolv.conf and then
   # calls vagrant up
-  #SSH from anywhere on the network (sshd)
-  config.vm.network :forwarded_port, guest: 22, host: 2222, host_ip: "127.0.0.1", id: "ssh", auto_correct: true
-  #In case somebody does not use "appserver" but "localhost"
-  config.vm.network :forwarded_port, guest: 80, host: 8888, id: "main"
-
+  #If you want to SSH from anywhere on the network (sshd) uncomment this
```

Comment:
Why not do this by default? Can it conflict with other applications?

Reply:
no, it's just that it is uncommon to want to be on a machine A and ssh to vagrant hosted on machine B